src/Docker/Client/Utils.hs

```diff
@@ -129,3 +133,16 @@ exclusionCheck f ps = any id (map (\(ExclusionPattern p) -> f ~~ T.unpack p) ps)
 inclusionCheck :: FilePath -> [InclusionPattern] -> Bool
 inclusionCheck f ps = any id (map (\(InclusionPattern p) -> f ~~ T.unpack p) ps)
 
+
+processLog :: Monad m => ConduitT BSS.ByteString BSS.ByteString m ()
+processLog = do
+        -- metadata (is the next string stdout or stderr)
```

Comment:
Do I understand correctly that the first 4 bytes in the STREAM just denote that the data marked as collected from stdout/stderr?

Reply:
As I explained in the issue, I couldn't find any documentation regarding the format. All I could find was this repo: https://github.com/mafintosh/docker-raw-stream

In that repo, the second 4 bytes represent the length of the subsequent string, and the first bit denotes whether the string is from stdout or stderr.

I assume that the first 4 bytes are reserved for metadata.

I have had the opportunity to test this branch and it works as expected.